<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="31421" started-at="2020-05-27T06:01:16Z" finished-at="2020-05-27T06:01:48Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="31421" started-at="2020-05-27T06:01:16Z" finished-at="2020-05-27T06:01:48Z">
      <class name="com.gurubanking.testcases.TC_LoginTest_001">
        <test-method status="PASS" signature="setupTest()[pri:0, instance:com.gurubanking.testcases.TC_LoginTest_001@46d56d67]" name="setupTest" is-config="true" duration-ms="4587" started-at="2020-05-27T11:31:16Z" finished-at="2020-05-27T11:31:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method status="PASS" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:com.gurubanking.testcases.TC_LoginTest_001@46d56d67]" name="loginTest" duration-ms="8111" started-at="2020-05-27T11:31:22Z" data-provider="test data" finished-at="2020-05-27T11:31:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr261116]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[JunUsud]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="FAIL" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:com.gurubanking.testcases.TC_LoginTest_001@46d56d67]" name="loginTest" duration-ms="5021" started-at="2020-05-27T11:31:30Z" data-provider="test data" finished-at="2020-05-27T11:31:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin_p1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.gurubanking.testcases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="FAIL" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:com.gurubanking.testcases.TC_LoginTest_001@46d56d67]" name="loginTest" duration-ms="7910" started-at="2020-05-27T11:31:35Z" data-provider="test data" finished-at="2020-05-27T11:31:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin_p2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.gurubanking.testcases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:com.gurubanking.testcases.TC_LoginTest_001@46d56d67]" name="loginTest" duration-ms="3833" started-at="2020-05-27T11:31:43Z" data-provider="test data" finished-at="2020-05-27T11:31:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr261116]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[JunUsud]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="closeTest()[pri:0, instance:com.gurubanking.testcases.TC_LoginTest_001@46d56d67]" name="closeTest" is-config="true" duration-ms="856" started-at="2020-05-27T11:31:47Z" finished-at="2020-05-27T11:31:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeTest -->
      </class> <!-- com.gurubanking.testcases.TC_LoginTest_001 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
